pydaikin/daikin_base.py

```diff
@@ -129,8 +133,9 @@ async def init(self):
         stop=stop_after_attempt(3),
         retry=retry_if_exception_type(
             (
-                ServerDisconnectedError,
                 ClientOSError,
+                ClientResponseError,
+                ServerDisconnectedError,
```

Comment:
```suggestion
                self.tenacy_retry
```
@kingy444, should these be set on class level, I guess we only need `ClientResponseError` for SkyFi.

Reply:
Unsure if much value in breaking it out to only skifi devices.

The other devices (as far as we know) aren't throwing these - but we could have just not had reports too.

If they aren't throwing them the retry doesn't matter

If they do throw them, a retry might also fix it for them 🤷